File: VendingMachineSpringMVC/src/main/webapp/js/item.js
/* 
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */money=0.0;
function addMoney(toAdd){

  money+=toAdd;
  $('#money-entered').val(money.toFixed(2));
}
$(document).ready(function(){
      



      $('#dollar-button').click(function(){ addMoney(1);});
      $('#quarter-button').click(function(){ addMoney(.25);});
      $('#dime-button').click(function(){ addMoney(.1);});
      $('#nickel-button').click(function(){ addMoney(.05);});
      $('#penny-button').click(function(){ addMoney(.01);});
      
      


});

function displayItemName(itemName){
 $('#itemName').val(itemName);
}


function clearAllForms(){
  $('#money-entered').val("");
  $('#message').val("");
  $('#itemName').val("");
  $('#change-output').val("");
}

File: VendingMachineSpringMVC/src/main/java/com/sg/vendingmachinespringmvc/model/Item.java
/*
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */
package com.sg.vendingmachinespringmvc.model;

import java.math.BigDecimal;

/**
 *
 * @author sabaaslam
 */
public class Item {
    private String itemName;
    private BigDecimal cost;
    private int inventory;
    private int itemId;

    public int getItemId() {
        return itemId;
    }

    public void setItemId(int itemId) {
        this.itemId = itemId;
    }


    public String getItemName() {
        return itemName;
    }

    public void setItemName(String itemName) {
        this.itemName = itemName;
    }

    public BigDecimal getCost() {
        return cost;
    }

    public void setCost(BigDecimal cost) {
        this.cost = cost;
    }

    public int getInventory() {
        return inventory;
    }

    public void setInventory(int inventory) {
        this.inventory = inventory;
    }
    
    
    

}
